libs/sdk-ui-tests-e2e/cypress/tools/table.ts

```diff
@@ -80,6 +82,25 @@ export class Table {
             .find(`.s-cell-${row}-${column}`);
     }
 
+    getPivotTableColumnTotalCell(row: number, column: number, match: boolean) {
+        return this.getElement()
+            .find(".ag-body-viewport")
+            .find(`.gd-total-column.s-cell-${row}-${column}`)
+            .find(".s-value")
+            .then(function ($elem) {
```

Comment:
is this .then() necessary? What about `.contains(nonEmptyValue).should(match ? "exist" : "not.exist")` (afaik contains should support regex)